shell/browser/api/electron_api_web_frame_main.h

```diff
@@ -139,6 +137,8 @@ class WebFrameMain final : public gin::Wrappable<WebFrameMain>,
   std::vector<content::RenderFrameHost*> Frames() const;
   std::vector<content::RenderFrameHost*> FramesInSubtree() const;
 
+  std::string LifecycleStateForTest();
```

Comment:
_(nit)_

These nits are pretty minor, especially since this isn't a user-visible method, but FWIW:

- can be `const`
- can return a `const char*` to match better with  `RenderFrameHostImpl::LifecycleStateImplToString()`?

```suggestion
  const char* LifecycleStateForTest() const;
```